BulkyWeb/Areas/Admin/Controllers/CategoryController.cs

```diff
@@ -0,0 +1,119 @@
+﻿using Bulky.DataAccess.Data;
+using Bulky.DataAccess.Repository.IRepository;
+using Bulky.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace BulkyWeb.Areas.Admin.Controllers
+{
+    [Area("Admin")]
+    public class CategoryController : Controller
+    {
+        private readonly IUnitOfWork _unitOfWork;
+        public CategoryController(IUnitOfWork unitOfWork)
+        {
+            _unitOfWork = unitOfWork;
+        }
+        public IActionResult Index()
+        {
+            try
+            {
+                List<Category> objCategoryList = _unitOfWork.Category.GetAll().ToList();
+                return View(objCategoryList);
+            }
+            catch (Exception ex)
+            {
+                TempData["error"] = "Category NotFound";
+                return RedirectToAction("Home/Index");
+            }
+        }
+
+        //[Route("Category/Create/{id}")]
+        public IActionResult Create()
+        {
+            try
+            {
+                return View();
+            }
+            catch (Exception ex)
+            {
+                TempData["error"] = "Category NotFound";
```

Comment:
same here you are throwing the same error everywhere even on creating a category. I would expect it to be "Could not create category"